extras/coverage-report.sh

```diff
@@ -1,7 +1,18 @@
-# Run tests, generate coverage report and open it on a browser
+#!/bin/sh
+# Run tests, generate coverage report and open it in a browser
 #
 # Requires: coverage 3.3 or above from http://pypi.python.org/pypi/coverage
 
-coverage run --branch $(which trial) --reporter=text tests
-coverage html -i
+trial="`which trial`"
```

Comment:
Right. But it's an enhancement on the status quo, where it doesn't run at all (for me).
If you dont't feel like merging the Makefile, I understand. But fixing this script is what got me started on this PR.

I don't know how to fix the script for python3, as it seems that the current way of running `coverage` is the reverse to this - as a plugin in the testsoftware - and making this script wholly obsolete.
I couldn't really figure out how to run coverage this way with tox or pytest.
Please advise?